files/ko/web/css/visual_formatting_model/index.md

```diff
@@ -47,7 +47,7 @@ CSS 선택기는 해당 상자에 이름을 부여하거나 스타일링을 할
 
 상자를 포함하는 블록은 인라인수준 상자 또는 블록수준 상자만을 포함합니다. 그러나 문서는 두 가지 모두를 혼합해 포함합니다. 그 경우 무명 블록 상자는 인접 인라인수준 상자 주변에 생성됩니다.
 
-### 예제
+### 예제 1
```

Comment:
Use numbers to differentiate

Reply:
- https://github.com/mdn/translated-content/pull/17716/commits/f99f49cd99791ab3f058aad13525eb24ad274b9a